specs/server.test.js

```diff
@@ -0,0 +1,57 @@
+var express = require('express');
+var request = require('supertest');
+var $ = require('jquery');
+
+var app = express();
```

Comment:
I think these tests cover more than enough of the back-end, but we should shift to the Front End and test as we build.